recipes/pyzmq/bld.bat

```diff
@@ -0,0 +1,13 @@
+set DISTUTILS_USE_SDK=1
+
+"%PYTHON%" setup.py configure --zmq=bundled
+if errorlevel 1 exit 1
+
+
+# For now, we build pyzmq with the bundled zeromq. There were a number of reasons for
+# this, including lib\libzmq.dll not being available, as well as
```

Comment:
# is an invalid comment character in bat files.  you need

REM comment
or
:: comment